src/core/Game.js

```diff
@@ -533,6 +533,7 @@ Phaser.Game = function (width, height, renderer, parent, state, transparent, ant
 * @property {boolean}            [GameConfig.alignV=false]                  - Sets {@link Phaser.ScaleManager#pageAlignVertically}.
 * @property {number|string}      [GameConfig.antialias=true]
 * @property {number|string}      [GameConfig.backgroundColor=0]             - Sets {@link Phaser.Stage#backgroundColor}.
+* @property {boolean}            [GameConfig.batchRender=false]             - Set to true if you want to flush the sprites until end of frame, to improve performance, especially on mobile.
```

Comment:
Can we change this comment to

`Set to true if you want to wait to flush draw calls until the end of the frame. This will improve performance, especially on mobile.`

Reply:
Lol yes the sentence made no grammatical sense either. Changed to:

Used by WebGL and if set to true then waits to flush the draw calls until end of frame, which should improve performance, especially on mobile.